### ISSUE TITLE
[Feature Request] Support for Daydream controller in the future?

### ISSUE THREAD

**Issue by ajayvarghese2000** [JibbSmart/JoyShockMapper#34]
I don't know how many different types of controllers you plan to support, if you do plan to support more than just the DS4 and Nintendo controllers, I would like to suggest adding in support for the Google Daydream controller. It would be great if we can repurpose our old Daydream controllers as an air mouse and such. I think someone has managed to get the gyro data off the daydream controllers but no one has managed to map the controls to keyboard and mouse keys.
https://mrdoob.github.io/daydream-controller.js/ 
https://github.com/mrdoob/daydream-controller.js/

**Comment by JibbSmart** [comment]
This would be really cool. Difficult to support without having one, though, and it's discontinued, isn't it?

**Comment by ajayvarghese2000** [comment]
Yeah it is discontinued the only way to get it now is buying it used of ebay or something, my whole setup is just sitting in storage at the moment it would be nice to repurpose it. If you do plan to support it I can mail you my controller depending on where in the world you are and if you're comfortable with me doing so.
